baseplate/frameworks/pyramid/__init__.py

```diff
@@ -294,3 +299,20 @@ def pshell_setup(env: Dict[str, Any]) -> None:
 
     """
     env["request"].start_server_span("shell")
+
+
+def get_is_healthy_probe(request: Request) -> int:
```

Comment:
can you add this to https://github.com/reddit/baseplate.py/blob/master/docs/api/baseplate/frameworks/pyramid.rst please?

Reply:
Done.